docs/api/qiskit/dev/qpy.mdx

```diff
@@ -172,7 +172,7 @@ These functions will raise a custom subclass of [`QiskitError`](exceptions#qiski
   Set the error message.
 </Class>
 
-### QPY\_VERSION
+### qiskit.qpy.QPY\_VERSION
```

Comment:
Do you know what these were before the docs redesign? I'm not certain it makes sense to use the full path here because it makes the header longer. I'm surprised the name is the full path and not only `QPY_VERSION`. It's not necessarily bad, I'm only curious why `name` was set to that

Reply:
Yes, they had the long `name` as well. [Here](https://github.com/Qiskit/documentation/blob/0406030c7e13a828bc5bd164b9ccb7aac29f4b06/docs/api/qiskit/dev/qpy.mdx?plain=1#L183) is the file before the docs redesign. I can investigate more about why it was set that way if you want :)